net/keepalived/Makefile

```diff
@@ -274,4 +274,103 @@ endif
 
 endef
 
+define Package/keepalived-sync
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Keepalived Master and Backup Synchronization
+  DEPENDS:= +keepalived +rsync +inotifywait +sudo +@BUSYBOX_CUSTOM +@BUSYBOX_CONFIG_TIMEOUT
```

Comment:
@jempatel I just noticed that you are activating `@BUSYBOX_CUSTOM` here to use `@BUSYBOX_CONFIG_TIMEOUT`. Can't we achieve this in another way?

Reply:
Can we not use the timeout of the [coreutils](https://github.com/openwrt/packages/blob/master/utils/coreutils/Makefile#L36) (coreutils-timeout).
I don't want to change the default settings of the busybox.
I will prepare a pullrequest for this.

Reply:
Yes, we can. I did not use it to avoid adding another additional package.